### DISCUSSION TITLE
Show full filepaths in HTML template

### DISCUSSION THREAD

**Discussion by b-d-e** [aquasecurity/trivy#7609] [Ideas]
### Description

For the template at path `contrib/html.tpl`, if there are multiple files scanned with the same name (e.g. multiple `Dockerfile`s in different folders, they all just get labelled "Dockerfile" (see picture).

It would be helpful for patching if the full filepaths were shown here. 
<img width="1285" alt="image" src="https://github.com/user-attachments/assets/98f62447-0d4f-471a-aa5a-87eb5e94d5ab">

### Target

Filesystem

### Scanner

Misconfiguration

**Comment by fatihtokus** [comment]
Hi @b-d-e ,

I have good news for you. There is a trivy plugin out there and it tackles this problem. If you are interested give it a try(https://github.com/aquasecurity/trivy/discussions/6980#discussioncomment-9885830):

trivy plugin install scan2html
trivy scan2html image docker.io/mattermost/mattermost-enterprise-edition:9.7.1 report.html
![image](https://github.com/user-attachments/assets/31ea29cf-628a-4fc5-9f48-14adfbc80a3c)

**Reply by b-d-e** [reply]
Thanks @fatihtokus! This is great! I'm giving it a try, but finding it's not quite reflective of the trivy output. From that, I get misconfigs detected, but when I run `trivy scan2html generate` these are not listed, only vulnerabilitise. Am I missing a flag for `generate`?

**Reply by fatihtokus** [reply]
I am glad you are using it @b-d-e. There are 2 different uses of plugin:

- use it while scanning i.e. trivy scan2html image docker.io/mattermost/mattermost-enterprise-edition:9.7.1 report.html
- use it to merge(not scan) already **generated** json results into one HTML report: trivy scan2html **generate** interactive_result.html from vulnerabilities.json misconfigs.json secrets.json

I hope this helps you. if not please share what commands you use so I can suggest
